docs/build/authentication.md

```diff
@@ -353,6 +353,104 @@ The keys returned by `exportKeyBundle` should be treated with the utmost care as
 </TabItem>
 </Tabs>
 
+## Start from private key
+
+You can start an XMTP from specific wallet key's if you want to maintain the same address.
```

Comment:
WDYT of changing this to something like this:

```
You can create an XMTP client with a private key using a compatible client library.
```

since JS/React only works with `viem` and `ethers` out of the box, i added the bit about a compatible client library.